components/NavigationBar.jsx

```diff
@@ -92,7 +92,7 @@ export default class NavigationBar extends React.Component {
                     options={applicationUrls.map((option) => {
                       return {
                         ...option,
-                        title: <span><b>Caseflow</b> {option.title}</span>
+                        title: <span><b>{option?.prefix || 'Caseflow'}</b> {option.title}</span>
```

Comment:
```suggestion
                        title: <span><b>{option.prefix || 'Caseflow'}</b> {option.title}</span>
```
I don't think we need optional chaining here since it should be a hash and it will just result in nil if it doesn't exist. Option.title is already being called and that will error anyhow if option isn't a hash for some reason. So it should either have a safe navigator in both spots or neither imo.